docs/v4/assets/index.html

```diff
@@ -852,13 +852,15 @@
       "type" : "number",
       "description" : "The purchase price of the asset",
       "format" : "double",
-      "example" : 1000.0000
+      "example" : 1000.0000,
+      "x-is-money" : true
```

Comment:
Hey @RettBehrens - at first glance i'm not sure this should be in the docs / field types.. This looks like (at least in the doc generation) it was adding the `x-vendor` extension as an actual field meta data?